src/store/reducers/appReducer.js

```diff
@@ -282,8 +284,10 @@ const updateSolutionSettingsReducer = (state, { payload, fromAction }) => update
 
 const updateActionSettingsReducer = (state, { payload, fromAction }) => {
   const { entities: { actionSettings } } = normalize(payload, actionSettingsListSchema);
+  const isDeployedAsOwner = actionSettings.Email.applicationPermissionsAssigned;
```

Comment:
nit: The name maybe better to be consistent with back-end to say 'applicationPermissionsAssigned' which is not assuming 'owner' to deploy this solution. it could be other similar admin role have permission to assign role for application.

Reply:
done